mappings/net/minecraft/entity/mob/PiglinBrain.mapping

```diff
@@ -23,7 +23,7 @@ CLASS net/minecraft/unmapped/C_mxpeolkc net/minecraft/entity/mob/PiglinBrain
 	FIELD f_pjfnnzxc PLAYER_HOLDING_LIKED_ITEM_MAX_LOOK_DISTANCE I
 	FIELD f_rukeoquj WALK_TO_ITEM_MAX_DISTANCE I
 	FIELD f_sedefqof BARTERING_ITEM Lnet/minecraft/unmapped/C_vorddnax;
-	FIELD f_srhqewhu DESIRED_DISTANCE_FROM_ENTITY_WHEN_AVOIDING I
+	FIELD f_srhqewhu AVOIDING_DESIRED_DISTANCE_FROM_ENTITY I
```

Comment:
Should it be AVOID or AVOIDING?

Reply:
Should be avoid imo: "avoid the desired distance from entity"
it's a constant and do not represent an ongoing action.

Reply:
Agreed

Reply:
AVOID_DESIRED_DISTANCE_FROM_ENTITY

Reply:
DESIRED_AVOIDANCE_DISTANCE?

Reply:
`DESIRED_ENTITY_AVOIDANCE_DISTANCE`?